app/controllers/admin/flags_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Admin::FlagsController < Admin::StaffController
```

Comment:
I think this could be `Admin::AdminController` for now, I don't think mods will be doing this. Later on if they do we could change it back.